### PR TITLE
Solidity test suite for M5

### PR DESCRIPTION
**Pull request by denalimarsh**
Fixes up the solidity test suite to reflect the latest changes

### CODE REVIEW

**Comment by denalimarsh** [comment]
We need to update the event name `Transfer` to `AppTransfer` because Transfer conflicts with the ERC20 token transfer event.

File: ethereum/contracts/ETHApp.sol [hunk 1]
```diff
@@ -14,7 +14,7 @@ contract ETHApp is Application {
     uint256 public nonce;
     uint256 public totalETH;
 
-    event Transfer(address _sender, bytes32 _recipient, uint256 _amount);
+    event AppTransfer(address _sender, bytes32 _recipient, uint256 _amount);
```
Comment (vgeddes) [inline ethereum/contracts/ETHApp.sol:17]: Ooh, we can't make this change without updating both the Relayer and Parachain. Is there a compelling reason why we should rename?

Comment (musnit) [reply ethereum/contracts/ETHApp.sol:17]: @vgeddes See above comment

Comment (vgeddes) [reply ethereum/contracts/ETHApp.sol:17]: Oh I see! Sorry I must have missed the other comment. @denalimarsh I'm gonna push the necessary parachain and relayer changes to this branch too.

**Review by vgeddes — APPROVED** [review]
LGTM :rocket:

**Pull request merged**